docs/release_process.md

```diff
@@ -0,0 +1,198 @@
+## Release Cadence
+
+Community EE is to be built and published on the next day and/or the day after ansible-core is released.
+
+## EE tag versioning
+
+Versioning would be the core tag + a patch number, e.g
+
+    - EE with core `2.16.2` comes out -> `community-ee:2.16.2-1`
+    - EE with core `2.16.3` comes out -> `community-ee:2.16.3-1`
+
+
+## Dependencies
+
+-   Podman/Docker
+
+##  Credentials
+
+- Access to  https://github.com/ansible-community/images repo
+- Access to ghcr.io
+
+
+## Build steps for Community-ee-base
+
+1. Go to the working directory.
+
+`cd images/execution-environments/community-ee-base`
+
+2. Create a new release branch
+
+`git checkout -b base-<ansible-core-version-base-ee-version>`
+
+3. Edit the `execution-environment.yml` file to the right `collection version` and `ansible-core version`. Get the correct `[ansible-core](https://pypi.org/project/ansible-core/)` version and `[ansible-<version>.deps](https://github.com/ansible-community/ansible-build-data/blob/main/9/ansible-9.0.1.deps)` and edit accordinly.
```

Comment:
Why should the `.deps` file be used here, instead of taking the latest versions of these collections from Ansible Galaxy?

Also if you put links in backticks, they won't work as links and will not be clickable, but show as code. Is that really intended?

Reply:
> Why should the `.deps` file be used here, instead of taking the latest versions of these collections from Ansible Galaxy?

To ensure collection versions are the correct ones which we know for sure that works. For example when we are releasing 2 versions of community ee 2.15 and 2.16 at the same time.

> Also if you put links in backticks, they won't work as links and will not be clickable, but show as code. Is that really intended?

No,  typo fixing it now.